### PR TITLE
Add iOS NetP pixels

### PR DESCRIPTION
**Pull request by quanganhdo**
Task/Issue URL: https://app.asana.com/0/72649045549333/1206057769488160/f

**Description**:

Add iOS NetP pixels for active/new user, VPN enabled success rate, tunnel failure/recovery, and latency.

**Steps to test this PR**:

1. Confirm pixel submission on Kibana
2. Make sure active user pixel includes the cohort value

**Copy Testing**:

* [ ] Use of correct apostrophes in new copy, ie `’` rather than `'`

**Orientation Testing**:

* [ ] Portrait
* [ ] Landscape

**Device Testing**:

* [ ] iPhone SE (1st Gen)
* [ ] iPhone 8
* [ ] iPhone X
* [ ] iPhone 14 Pro
* [ ] iPad

**OS Testing**:

* [ ] iOS 14
* [ ] iOS 15
* [ ] iOS 16

**Theme Testing**:

* [ ] Light theme
* [ ] Dark theme

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: Core/PixelEvent.swift [hunk 1]
```diff
@@ -851,8 +863,15 @@ extension Pixel.Event {
 
         case .networkProtectionActiveUser: return "m_netp_daily_active_d"
         case .networkProtectionNewUser: return "m_netp_daily_active_u"
+        case .networkProtectionEnableAttemptConnecting: return "m_netp_ev_enable_attempt"
+        case .networkProtectionEnableAttemptSuccess: return "m_netp_ev_enable_attempt_success"
+        case .networkProtectionEnableAttemptFailure: return "m_netp_ev_enable_attempt_failure"
+        case .networkProtectionTunnelFailureDetected: return "m_netp_ev_tunnel_failure"
+        case .networkProtectionTunnelFailureRecovered: return "m_netp_ev_tunnel_failure_recovered"
+        case .networkProtectionLatency(let quality): return "m_netp_ev_\(quality.rawValue)_latency"
+        case .networkProtectionLatencyError: return "m_netp_ev_latency_error_d"
```
Comment (quanganhdo) [inline Core/PixelEvent.swift:872]: Whether or not to include the `_d` suffix depends solely on which pixel firing method is picked. I'd clean that up later.

File: Core/PixelEvent.swift [hunk 2]
```diff
@@ -851,8 +863,15 @@ extension Pixel.Event {
...
         case .networkProtectionRekeyCompleted: return "m_netp_rekey_completed"
-        case .networkProtectionLatency: return "m_netp_latency"
+        case .networkProtectionEnabledOnSearch: return "m_netp_enabled_on_search"
```
Comment (quanganhdo) [inline Core/PixelEvent.swift:874]: Not yet reported in this PR.

File: Core/PixelEvent.swift (hunk 1 above)
Comment (samsymons) [reply Core/PixelEvent.swift:872]: :+1:

**Review by samsymons — APPROVED** [review]
Each of these looks good in my testing - thanks!

**Pull request merged**